### PR TITLE
Begin new Working Group Draft version

### PR DESCRIPTION
**Pull request by y-guyon**
***
<a href="https://pr-preview.s3.amazonaws.com/y-guyon/av1-avif/pull/207.html" title="Last updated on Jun 26, 2024, 1:41 PM UTC (a8471f7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/207/77bd20d...y-guyon:a8471f7.html" title="Last updated on Jun 26, 2024, 1:41 PM UTC (a8471f7)">Diff</a>

### CODE REVIEW

**Review by wantehchang — APPROVED** [review]
It would be good to mention in the commit message that the (unused?) text macto `SPECVERSION` is deleted.

File: index.bs [hunk 1]
```diff
@@ -1,17 +1,16 @@
 <pre class='metadata'>
 Group: AOM
-Status: FD
-Text Macro: SPECVERSION v1.1.0
```
Comment (cconcolato) [inline index.bs:4]: You could also keep the SPECVERSION metadata and set it directly to v1.2.0

File: index.bs [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
+Status: WGD
 Title: AV1 Image File Format (AVIF)
-URL: https://AOMediaCodec.github.io/av1-avif/v1.1.0.html
+URL: https://AOMediaCodec.github.io/av1-avif
 !Latest version: <a href="https://AOMediaCodec.github.io/av1-avif">https://AOMediaCodec.github.io/av1-avif</a>
-!Previously approved version: <a href="v1.0.0.html">v1.0.0</a>
+!Previously approved version: <a href="v1.1.0.html">v1.1.0</a>
```
Comment (cconcolato) [inline index.bs:7]: Can you align the headers with the ones in AV1-ISOBMFF https://aomediacodec.github.io/av1-isobmff/ 
```
This version:
https://aomediacodec.github.io/av1-avif/
Previously approved version:
https://aomediacodec.github.io/av1-avif/v1.1.0.html
Latest approved version:
https://aomediacodec.github.io/av1-avif/latest-approved.html
Latest draft version:
https://aomediacodec.github.io/av1-avif/latest-draft.html
Test Suite:
https://aomediacodec.github.io/av1-isobmff/conformance/
Issue Tracking:
GitHub
```
The latest-approved.html and latest-draft.html should be symlinks to the corresponding versions.

File: index.bs [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
 Shortname: av1-avif
 Editor: Cyril Concolato, Netflix, cconcolato@netflix.com
```
Comment (cconcolato) [inline index.bs:9]: Move me to the list of former editors and add the new editors.

**Review by cconcolato — APPROVED** [review]

File: index.bs (hunk 3 above)
Comment (y-guyon) [reply index.bs]: Not sure who to put here so I just added myself for now. Contributors to be added on the next pull requests.

**Comment by cconcolato** [comment]
LGTM

**Pull request merged**